DuplicateDetective/src/main/java/duplicates/Driver.java

```diff
@@ -0,0 +1,48 @@
+package duplicates;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Driver {
+    static Scanner IN = new Scanner(System.in);
+
+    public static void main(String[] args) {
```

Comment:
A small architectural note.

I suggest to extract `main` to a new class named "Application" or "DuplicateDetective" - so that an untrained eye can quickly find it.